core/templates.yml

```diff
@@ -20,7 +20,7 @@
 # this file. For more information, see the Shotgun Pipeline Toolkit Documentation.
 
 
-
+include: ../env/includes/unreal/templates.yml
```

Comment:
Just a friendly comment/question. Have you consider putting this file inside core folder?

Reply:
It's outside of `core` on purpose so all unreal files are self contained into a single folder. Which makes updated from default2 less prone to conflicts.